.gitignore

```diff
@@ -1,3 +1,4 @@
+playground.py
```

Comment:
Hi, I prefer not to touch this file for now, not sure how this is usually handled though.

Reply:
No problem. After all reviews are done, I will do last update just to take it down.